plugins/modules/system/parted.py

```diff
@@ -95,6 +95,12 @@
     type: str
     choices: [ absent, present, info ]
     default: info
+  fs_type:
+    description:
+     - If specified, will set filesystem type to given partition
+    type: str
+    default: ext4
+    version_added: '2.10'
```

Comment:
```suggestion
    version_added: '2.10'
```
@felixfontein do we have any official decision what we should put instead?

Reply:
No. Let's keep `'2.10'` for now.

Reply:
clear, thanks!